library/spdm_requester_lib/libspdm_req_handle_error_response.c

```diff
@@ -241,18 +241,15 @@ libspdm_return_t libspdm_handle_error_large_response(
     transport_header_size = spdm_context->transport_get_header_size(spdm_context);
 
     libspdm_get_scratch_buffer(spdm_context, (void**)&scratch_buffer, &scratch_buffer_size);
-    LIBSPDM_ASSERT(
-        scratch_buffer_size >=
-        LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE + LIBSPDM_SCRATCH_BUFFER_LARGE_MESSAGE_CAPACITY);
+    LIBSPDM_ASSERT(scratch_buffer_size >= LIBSPDM_SCRATCH_BUFFER_SIZE);
 
-    /* The first LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE bytes of the scratch
-     * buffer may be used for other purposes. Use only after that section. */
+    /* The large response should be stored in a special region of the scratch buffer */
     large_response = scratch_buffer + LIBSPDM_SCRATCH_BUFFER_LARGE_MESSAGE_OFFSET;
     large_response_capacity = LIBSPDM_SCRATCH_BUFFER_LARGE_MESSAGE_CAPACITY;
 
     /* Temporary send/receive buffers for chunking are in the scratch space */
-    message = scratch_buffer + LIBSPDM_SCRATCH_BUFFER_SENDER_RECEIVER_OFFSET;
-    message_size = LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE;
+    message = scratch_buffer + LIBSPDM_SCRATCH_BUFFER_CHUNKING_SENDER_RECEIVER_BUFFER_OFFSET;
+    message_size = LIBSPDM_SCRATCH_BUFFER_CHUNKING_SENDER_RECEIVER_BUFFER_SIZE;
```

Comment:
As the comments, this buffer (scratch_buffer + LIBSPDM_SCRATCH_BUFFER_CHUNKING_SENDER_RECEIVER_BUFFER_OFFSET) is temp send/receive buffer. Want to make sure it will be copied to SENDER_RECEIVE when it is sent to transport_io spdm_context->send_message/spdm_context->receive_message.